wasm/src/programs/verifying_key/credits.rs

```diff
@@ -134,15 +134,15 @@ impl VerifyingKey {
     /// @returns {VerifyingKey} Verifying key for the unbond_delegator_as_delegator function
     #[wasm_bindgen(js_name = "unbondPublicVerifier")]
     pub fn unbond_public_verifier() -> VerifyingKey {
-        VerifyingKey::from_bytes(&snarkvm_parameters::testnet3::UnbondPublicVerifier::load_bytes().unwrap()).unwrap()
+        VerifyingKey::from_bytes(&snarkvm_parameters::testnet::UnbondPublicVerifier::load_bytes().unwrap()).unwrap()
     }
 
     /// Returns the verifying key for the bond_public function
     ///
     /// @returns {VerifyingKey} Verifying key for the bond_public function
     #[wasm_bindgen(js_name = "isBondPublicVerifier")]
     pub fn is_bond_public_verifier(&self) -> bool {
-        self == &VerifyingKey::from_bytes(&snarkvm_parameters::testnet3::BondPublicVerifier::load_bytes().unwrap())
+        self == &VerifyingKey::from_bytes(&snarkvm_parameters::testnet::BondPublicVerifier::load_bytes().unwrap())
```

Comment:
This is a good start, though we'll need to support mainnet soon as well. I'm fine focusing on only testnet for now and adding mainnet support in a future PR

Reply:
Yes, I will support mainnet in another PR, that's a much bigger change.